### PR TITLE
726-dpl-639-restrict-values-in-the-schema-of-priority-flags-to-booleans

### PR DESCRIPTION
**Pull request by yoldas**
Closes #

Changes proposed in this pull request:

* Use newer version of lab-share-lib to enforce schema

### CODE REVIEW

**Comment by yoldas** [comment]
We used boolean instead of null boolean union type as the required JSON formatting with type info to validate while reading looks non-intuitive.

**Review by emrojo — DISMISSED** [review]

File: Pipfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ flask = "~=2.2"
 flask-apscheduler = "~=1.12"
 gunicorn = "~=20.1"
 more-itertools = "~=9.1"
-mysql-connector-python = "~=8.0"
+mysql-connector-python = "==8.0.31"
```
Comment (sdjmchattie) [inline Pipfile:25]: You shouldn't need to pin this here.  I think something you ran locally updated the version being installed in the `Pipfile.lock` which is the actual log of which versions to install.  Having 8.0.31 installed satisfies the `"~=8.0"` in this `Pipfile` so there's no need for the pin.

File: dependencies/scripts/mongo-init-replica-set.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 mongo <<EOF
 var config = {
     _id: "heron_rs",
-    members: [{ _id: 0, host: "mongo:27017"}]
+    members: [{ _id: 0, host: "127.0.0.1:27017"}]
```
Comment (sdjmchattie) [inline dependencies/scripts/mongo-init-replica-set.sh:6]: You'll likely find that changing this stops the Docker stack from initialising the replica set properly.

File: Pipfile (hunk 1 above)
Comment (yoldas) [reply Pipfile]: I have unpinned mysql-connector-python in Pipfile.

File: dependencies/scripts/mongo-init-replica-set.sh (hunk 2 above)
Comment (sdjmchattie) [reply dependencies/scripts/mongo-init-replica-set.sh:6]: Actually no I'm wrong.  The port is exposed to the local host.  It has been working for me in the previous format though so I am not sure why you've needed to change it

Comment (yoldas) [reply dependencies/scripts/mongo-init-replica-set.sh:6]: I am running the services in docker container but tests locally. The hostname mongo cannot be resolved. Changing hostname to loopback seemed harmless as the port is exposed as you said.

**Comment by stevieing** [comment]
Closed as being on hold.

**Pull request closed without merging**